### ISSUE TITLE
[Accessibility] div.monaco-tokenized-source requires aria-live="assertive"

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#12496]
CC -isidorn and -meganrogge

-- Current Issue

1. Load NVDA or any screen reader.

1. Launch VSCode.

1. enable copilot extension.

1. Create any ts or js file.

1. Try to trigger copilot-powered inline suggestion (`Alt+\`).

1. See (hear) if suggestion pop-up is automatically read: it's not unless you press `Ctrl+K, Ctrl+I` to explicitly move your focus to the tooltip area.

* Notes: This is because `"div.monaco-tokenized-source"` does not have `aria-live="assertive"` to announce the dymanic content to screen reader.

* Notes: copilot is just one of the examples. Any extensions or widgets that depend `"div.monaco-tokenized-source"` have the same issue.

-- Expected Behavior

The suggested dynamic content needs to be automatically announced to screen reader users.

* Note: I am willing to fix this issue and test with my screen reader.

-- System Info

Version: 1.74.0-insider (system setup)
Commit: d0e44be43287acadb13f7c2808332e00fe52568d
Date: 2022-11-25T08:00:08.922Z
Electron: 19.1.6
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22621
Sandboxed: Yes
